docs/configurations.md

```diff
@@ -258,3 +262,14 @@ scalar.db.contact_points=indirect:<SCALARDB_CLUSTER_CONTACT_POINT>
 ```
 
 For details about client configurations, see the ScalarDB Cluster [client configurations (redirects to the Enterprise docs site)](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/developer-guide-for-scalardb-cluster-with-java-api/#client-configurations).
+
+### Configuration example #3 - Use placeholders
```

Comment:
Revised this heading to be a noun phrase since the other examples follow a noun phrase pattern.

```suggestion
### Configuration example #3 - Placeholders usage
```